fl4health/clients/fed_prox_client.py

```diff
@@ -96,7 +94,21 @@ def set_parameters(self, parameters: NDArrays, config: Config) -> None:
             initial_layer_weights.detach().clone() for initial_layer_weights in self.model.parameters()
         ]
 
-    def compute_loss(self, preds: Dict[str, torch.Tensor], target: torch.Tensor) -> Losses:
+    def compute_loss(
+        self, preds: Dict[str, torch.Tensor], features: Dict[str, torch.Tensor], target: torch.Tensor
+    ) -> Losses:
+        """
+        Computes loss given predictions of the model and ground truth data.
```

Comment:
Maybe add a description that we're also adding in the proximal loss, comparing the l2 norm between the initial and final weights of local training